de_web_plugin_private.h

```diff
@@ -200,23 +203,25 @@
 #define WRITE_USERTEST          (1 << 16)
 
 // manufacturer codes
-#define VENDOR_ATMEL    0x1014
-#define VENDOR_JENNIC   0x1037
-#define VENDOR_DDEL     0x1135
-#define VENDOR_INNR     0x1166
-#define VENDOR_INNR2    0x1168
-#define VENDOR_INSTA    0x117a
-#define VENDOR_IKEA     0x117c
-#define VENDOR_NYCE     0x10b9
-#define VENDOR_PHILIPS  0x100B
-#define VENDOR_OSRAM_STACK  0xBBAA
-#define VENDOR_OSRAM    0x110C
-#define VENDOR_CENTRALITE   0x104e
-#define VENDOR_UBISYS   0x10F2
+#define VENDOR_NONE         0x0000
+#define VENDOR_ATMEL        0x1014
+#define VENDOR_JENNIC       0x1037
+#define VENDOR_DDEL         0x1135
+#define VENDOR_INNR         0x1166
+#define VENDOR_INNR2        0x1168
+#define VENDOR_INSTA        0x117A
+#define VENDOR_IKEA         0x117C
+#define VENDOR_1002         0x1002     // @manup, where can I lookup manufacturer code?
+#define VENDOR_PHILIPS      0x100B
+#define VENDOR_CENTRALITE   0x104E
+#define VENDOR_NYCE         0x10B9
+#define VENDOR_UBISYS       0x10F2
+#define VENDOR_BEGA         0x1105
+#define VENDOR_OSRAM        0x110C
 #define VENDOR_BUSCH_JAEGER 0x112E
-#define VENDOR_BEGA 0x1105
-#define VENDOR_PAULMANN 0x119d
-#define VENDOR_NONE   0x0000
+#define VENDOR_PAULMANN     0x119D
+#define VENDOR_120B         0x120B     // @manup, where can I lookup manufacturer code?
```

Comment:
Wireshark source code contains  a comprehensive list:

http://cgit.osmocom.org/wireshark/plain/epan/dissectors/packet-zbee.h

Reply:
Found 1 out of 2...